### ISSUE TITLE
sandbox activations 

### ISSUE THREAD

**Issue by BarnetMichal** [allegro/allegro-api#7718]
Hello, I would like to request sales activation on my sandbox account.
Tried to figure out the way to do it, but it stays on 'We are verifying the bank account details'
Here is my client email: michal.b@koongo.com
I would like to ask what is the recommended way to create products. Via the 'sale/product-proposal' endpoint or 'sale/product-offers'?  'product-offers' has a definition idType": "GTIN" which 'product-proposal' does not. Is there any difference between ednpoints?
Thank you in advance!

**Comment by PrzemyslawLukanowski** [comment]
Hello,

I’ve activated your account.

Yes, there is some difference between these endpoints. You always need to use /sale/product-offers to create an [offer](https://developer.allegro.pl/tutorials/glossary-LRVjol8kdCA#offer). Each offer needs to be associated with a [product](https://developer.allegro.pl/tutorials/glossary-LRVjol8kdCA#product) - either from our Catalog (you can check whether the product exists using [GET /sale/products](https://developer.allegro.pl/tutorials/list-offer-assigned-product-one-request-D7Kj9M71Bu6#how-to-find-a-product)) or you can create a new product within the same request. This process is described in [our tutorial](https://developer.allegro.pl/tutorials/list-offer-assigned-product-one-request-D7Kj9M71Bu6#how-to-list-an-offer-with-a-product).  Worth noting fact is, that by using this resource you can upload images from external servers, you don't have to send them to Allegro server first. The /sale/product-proposals resource is intended only to create products, so after you create a product you need to use /sale/product-offers anyway to associate this product with your offer. [Here](https://developer.allegro.pl/tutorials/list-offer-assigned-product-one-request-D7Kj9M71Bu6#how-to-create-a-product) you can find more information about product-proposals endpoint.

If you want change language on website to English, please use the icon on the top-right corner.

**Comment by BarnetMichal** [comment]
thanks for activating the sandbox. 

Regarding the product creation, is it possible to try this somewhere? I don't have permission for this on the sandbox. Is there a test store on the production server to test product creation? 
Thank you.

**Comment by PrzemyslawLukanowski** [comment]
You should be able to create products on Sandbox. I see that the 403 Forbidden error was returned yesterday when your account was not activated, but right now, it should work properly.

**Comment by BarnetMichal** [comment]
You`re right, I`m able to create products now, thanks. 

I have a new issue, I tried to register a new app for our production account team@koongo.com and after I hit the Register button, nothing happened, It just set the green app name field, see screen. 
![Capture](https://github.com/allegro/allegro-api/assets/26611033/a8f7f594-b4ca-473b-8c36-4a7535203718)

Can you please help me with that?

**Comment by PrzemyslawLukanowski** [comment]
When you enter this site - https://apps.developer.allegro.pl/, do you see any additional warnings? If not, please try to clear cookies or use another browser. If that doesn't help, please take a screenshot of what the console returns after clicking the button.

**Comment by BarnetMichal** [comment]
I logged out but then got banned when I logged back in, see

You've been blocked.
Why do we need this lock? Something strange in your browser's behaviour has caught our attention.

There are several possible explanations for this:
You're surfing the web and clicking at faster than humanly possible speeds.
something on your computer is preventing Javascript from working properly
or there is a robot that is on the same network as you (IP 95.168.200.121)

My console screenshot
![Capture](https://github.com/allegro/allegro-api/assets/26611033/12b18f0b-6658-40ea-8bd5-1f3518cc5633)

I tried to create the app with a different browser, see the screenshot
![Capture](https://github.com/allegro/allegro-api/assets/26611033/b1d08d82-ebcb-43a3-84b0-2e49c46e53ae)

**Comment by PrzemyslawLukanowski** [comment]
Thank you for that information. I've forwarded this issue to the appropriate team and will get back when I have more details, please be patient.

**Comment by PrzemyslawLukanowski** [comment]
@BarnetMichal Did you have the console open when you logged out/logged in? If so, that will be the case why you have been blocked.

**Comment by BarnetMichal** [comment]
yeah, I had. I opened it due to cookies clearing and console screens and forgot to close it.
I`m unblocked now and I managed to create the app. Thanks for the help.

**Comment by PrzemyslawLukanowski** [comment]
No problem, thank you for the confirmation.
